Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM python: latest
 
-COPY main.py binanceinfo.py Telegramsender.py .gitignore
+COPY main.py binanceinfo.py Telegramsender.py
 
-CMD ["python", "binanceinfo.py", "Telegramsender.py", ".gitignore",]
\ No newline at end of file
+CMD ["python", "./binanceinfo"]
```

Comment:
You will run `binanceinfo.py` or `main.py` ? How do you launch your code locally ?